ui/app/src/views/ViewDashboard.tsx

```diff
@@ -128,7 +128,6 @@ function ViewDashboard() {
           onSuccess: (createdDashboard: DashboardResource) => {
             actionRef.current = undefined;
             successSnackbar(`Dashboard ${getDashboardDisplayName(createdDashboard)} has been successfully created`);
-            navigate(`/projects/${createdDashboard.metadata.project}/dashboards/${createdDashboard.metadata.name}`);
```

Comment:
fyi this was basically never called; when `navigate()` call is placed here in this promise, the auto redirection caused by the form submit is taking precedence.